aptos-move/framework/aptos-framework/sources/SimpleToken.move

```diff
@@ -0,0 +1,102 @@
+/// This exists to provide convenient access to Tokens within Aptos for folks that do not want
+/// additional features within the metadata.
+module AptosFramework::SimpleToken {
+    use AptosFramework::Token;
+    use AptosFramework::TokenTransfers;
+
+    struct NoMetadata has copy, drop, store { }
```

Comment:
This should be SimpleToken instead of NoMetadata

Reply:
Will update!